compiler-rt/lib/mpk_untrusted/mpk_common.h

```diff
@@ -32,4 +34,12 @@ extern std::atomic<uint64_t> AllocSiteCount;
   } while (0)
 #endif
 
+// SINGLE_REPORT functions as the macro above but is intended
```

Comment:
Why not just call `sanitizer::Report()` directly?

Reply:
I like this little macro because I can turn on any of my currently existing `REPORT` macros by adding the prefix to it. If you want me to remove it and just use it for myself I am perfectly happy to.